### ISSUE TITLE
About FeatUp for Semantic segmentation

### ISSUE THREAD

**Issue by innovation-x** [mhamilton723/FeatUp#19]
I think the open source only offered the demos for Class Activation Maps. Can you share the usage for semantic segmentation and depth estimation as well?

**Comment by ShadowBbBb** [comment]
Same question about the usage for depth estimation!

**Comment by mhamilton723** [comment]
Yes we are working to make our probes ready for public usage thanks for your patience!

**Comment by ahmadmustafaanis** [comment]
Hi @mhamilton723 
ANY updates on it? Can I use it for depth?

**Comment by Billy2001-shadow** [comment]
mark
